src/main/java/org/sagebionetworks/web/client/widget/entity/menu/v3/EntityActionMenuProps.java

```diff
@@ -0,0 +1,53 @@
+package org.sagebionetworks.web.client.widget.entity.menu.v3;
+
+import java.util.Map;
+import jsinterop.base.JsPropertyMap;
+import org.sagebionetworks.web.client.jsinterop.entity.actionmenu.ActionConfiguration;
+import org.sagebionetworks.web.client.jsinterop.entity.actionmenu.ActionConfigurationMap;
+import org.sagebionetworks.web.client.jsinterop.entity.actionmenu.EntityActionMenuDropdownMap;
+import org.sagebionetworks.web.client.jsinterop.entity.actionmenu.EntityActionMenuLayout;
+import org.sagebionetworks.web.client.jsinterop.entity.actionmenu.EntityActionMenuPropsJsInterop;
+
+public class EntityActionMenuProps {
```

Comment:
POJO that just wraps the EntityActionMenuPropsJsInterop class, but is easier to test; we don't have to deal with restrictions that come with using JsInterop without a browser.